app/views/membership_applications/_membership_application_fields.html.haml

```diff
@@ -5,11 +5,15 @@
       = f.label :membership_number, t('membership_applications.show.membership_number'), class: 'required'
       = f.text_field :membership_number, class: 'wpcf7-form-control'
 
-    = f.label :first_name, t('membership_applications.show.first_name'), class: 'required'
-    = f.text_field :first_name, class: 'wpcf7-form-control'
 
-    = f.label :last_name, t('membership_applications.show.last_name'), class: 'required'
-    = f.text_field :last_name, class: 'wpcf7-form-control'
+    = f.fields_for :user do |ff|
+      = ff.label :first_name, t('membership_applications.show.first_name'), class: 'required'
+      = ff.text_field :first_name, class: 'wpcf7-form-control'
+
+
+      = ff.label :last_name, t('membership_applications.show.last_name'), class: 'required'
+      = ff.text_field :last_name, class: 'wpcf7-form-control'
```

Comment:
I don't think we want to support editing these user attributes in this view.  Currently, we user `current_user` as the user to whom the MA will be associated.  The only reason to also collect these user attributes would be if we allowed an admin to create a MA on behalf of another user ..... but that would be another story.

We _will_ be collecting/editing these attributes from a "user profile" create/edit form - but that is also another story.